### PR TITLE
Adding descriptions to multiple pages. 

### PR DESCRIPTION
**Pull request by RoAnnC1**
Minor updates to remove extra spacing, cap headings, add SAP, etc.

Deleted commented out text.

### CODE REVIEW

File: src/api-reference/expense/quick-expense/v3.quick-expense.markdown [hunk 1]
```diff
@@ -160,65 +150,57 @@ Name|Type|Format|Description
...
 
-
 ### Request URL
 ```
 https://www.concursolutions.com/api/v3.0/expense/quickexpenses/gWr7QiDXTuWSO8bsFmK87bnP%24sC8fZmg
 ```
 
-
-### JSON example of a successful response
+### JSON Example of a Successful Response
```
Comment (sarraloew) [inline src/api-reference/expense/quick-expense/v3.quick-expense.markdown]: Let's remove this one since it has no content

**Review by sarraloew — CHANGES_REQUESTED** [review]

File: src/api-reference/expense/quick-expense/v3.quick-expense.markdown (hunk 1 above)
Comment (RoAnnC1) [reply src/api-reference/expense/quick-expense/v3.quick-expense.markdown]: Fixed, I thought I got them all.

**Review by sarraloew — APPROVED** [review]

**Comment by sarraloew** [comment]
:shipit:

**Comment by charlieconcur** [comment]
:shipit:

**Pull request merged**